src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -5,18 +5,30 @@ import (
 	"net"
 	"net/http"
 	"reflect"
+	"strings"
 	"sync"
 	"time"
 
 	"github.com/armon/go-socks5"
 	"github.com/getlantern/balancer"
+	"github.com/getlantern/detour"
 	"github.com/getlantern/eventual"
 	"github.com/getlantern/golog"
 )
 
+const (
+	// LanternSpecialDomain is a special domain for use by lantern that gets
+	// resolved to localhost by the proxy
+	LanternSpecialDomain          = "ui.lantern.io"
```

Comment:
Just a quick note: We were advertising getlantern.org for a long time, would some domain like `start.getlantern.org` or `launch.getlantern.org` be more familiar to user and help to strengthen the brand?